library/notary

```diff
@@ -1,8 +1,12 @@
 # maintainer: David Lawrence <david.lawrence@docker.com> (@endophage)
 
-server: git://github.com/docker/notary-official-images@5fbf22ada9e5167fe4e9340ac947e8b3110aceba notary-server
+server: git://github.com/docker/notary-official-images@v0.5.0 notary-server
+server-0.5.0: git://github.com/docker/notary-official-images@v0.5.0 notary-server
+signer: git://github.com/docker/notary-official-images@v0.5.0 notary-signer
+signer-0.5.0: git://github.com/docker/notary-official-images@v0.5.0 notary-signer
+server-0.4.2: git://github.com/docker/notary-official-images@v0.4.2 notary-server
+signer-0.4.2: git://github.com/docker/notary-official-images@v0.4.2 notary-signer
```

Comment:
Yes and no. We specifically message "use notary version X with docker version Y" and that includes server and signer. We release all 3 notary components together and everything else we do, the images we deploy to production, the notary client binary, and our vendoring into other docker projects, is done using the same semantic tag. 

At the end of the day it's kind of meaningless anyway. The docker/notary-official-image repo just contains opaque binaries with no tie-back to the source they were build from.